esx_service/vmdk_ops.py

```diff
@@ -1711,6 +1712,101 @@ def wait_for_tasks(si, tasks):
 
 #------------------------
 
+def start_vm_changelistener():
+    """
+    Listen to power state changes of VMs running on current host
+    """
+    si = get_si()
+    pc = si.content.propertyCollector
+    create_vm_powerstate_filter(pc, si.content.rootFolder)
```

Comment:
What do we do if it throw a exception  when creating the filter spec in this function?